src/modules/entities/modules/reports/services/sagas/fetchReports.js

```diff
@@ -23,7 +23,7 @@ async function resolveData(documentSnapshot) {
 }
 
 function* fetchReports(action) {
-    const limit = 20;
+    const limit = 18;
```

Comment:
this should be outside of this function as global const
=> const LIMIT = 18;
you don't need to redeclare it every time this is called

Reply:
Done